doc/user-guide/secrets.md

```diff
@@ -0,0 +1,43 @@
+# Secrets
+
+```{tip}
+You can safely store API keys as secrets as all values are encrypted.
+```
+
+If your application requires secrets, you can define them when
+deploying it. To expand the section click on the triangle to the left:
+
+
+![](../static/secrets/secrets-section.png)
+
+
+Then, add the secrets keys and values:
+
+![](../static/secrets/secrets-sample.png)
+
+To add more, click on the button below:
+
+![](../static/secrets/secrets-plus.png)
+
+
+```{tip}
+You can also define secrets using the [command-line interface.](./cli.md)
+```
+
+## Reading secrets
+
+To read the secrets, use the following Python code:
+
+```python
+from os import environ
+
+value = environ.get("MY_SECRET")
+
+if value is None:
+    print("MY_SECRET is undefined!")
+else:
+    print(f"MY_SECRET: {value}")
+```
+
+
+A complete example is available [here.](https://github.com/ploomber/doc/blob/main/examples/voila/env-variable/app.ipynb)
```

Comment:
Do we need to rename this project also to `secrets`? And maybe change `value = environ.get("ENV_VAR", None)` to `value = environ.get("SECRET", None)`

Reply:
agree

Reply:
[renamed the project and fixed the link](https://github.com/ploomber/doc/pull/171/files#diff-8f7b69d5cb7da4e9386d20f6b5c3645123a4d09eacba1fd8d168e164e0cdaf8cR43)